docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
 
   postgres:
-    image: postgres:13.2
+    image: postgres:12.11
```

Comment:
Changing the defined image for the php service is pretty far out of scope for the original issue.  I was noticing that most of our workflows run in "ubuntu-latest" so I'll make an issue to lock down the images, including the php one to bullseye.

Our "db client" is pdo_pgsql, right?  Does it actually use psql or libpq underneath?  I couldn't find anything about that when I checked.

Reply:
Added #4102

Reply:
Thanks!

> Changing the defined image for the php service

But not a change, just locked down so that postgres libpq pkg doesn't silently change versions when a new debian base image arrives (with a diff version) :)